### PR TITLE
update few Feature

### PR DESCRIPTION
**Pull request by ehlxr**
## PR Checklist
**Please check if your PR fulfills the following requirements:**

- [ ] The commit message follows [our guidelines](https://github.com/theme-next/hexo-theme-next/blob/master/.github/CONTRIBUTING.md).
- [x] Tests for the changes have been added (for bug fixes / features).
   - [x] Muse | Mist have been tested.
   - [x] Pisces | Gemini have been tested.
- [ ] Docs have been added / updated (for bug fixes / features).

## PR Type
**What kind of change does this PR introduce?**  

- [ ] Bugfix.
- [x] Feature.
- [ ] Code style update (formatting, local variables).
- [ ] Refactoring (no functional changes, no api changes).
- [ ] Build related changes.
- [ ] CI related changes.
- [ ] Documentation content changes.
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number(s): N/A

## What is the new behavior?
* Update picture display
* add gitalk support
* update sidebar author image display

### How to use?
In NexT `_config.yml`:
```yml
gitalk:
  enable: true
  click_show: false # if true,need to click to show
  owner: # MUST HAVE, Your Github Username
  repo: # MUST HAVE, The name of the repo you use to store gitalk comments
  client_id: # MUST HAVE, Github client id for the gitalk
  client_secret: # EITHER this or proxy_gateway, Github access secret token for the gitalk
  admin: # MUST HAVE, Your Github Username
```

## Does this PR introduce a breaking change?
- [ ] Yes.
- [x] No.

### CODE REVIEW

**Comment by ehlxr** [comment]
Update picture display
add gitalk support
update sidebar author image display

**Comment by wafer-li** [comment]
What do you mean **update**? 

Show some demo or screenshots of it.

And for the gitalk, add something like the https://github.com/theme-next/hexo-theme-next/pull/58#issuecomment-358982728 described. Or we will not approve it.

File: layout/_layout.swig [hunk 1]
```diff
@@ -93,5 +93,25 @@
   {% include '_third-party/scroll-cookie.swig' %}
   {% include '_third-party/exturl.swig' %}
   {% include '_third-party/bookmark.swig' %}
+
+    <script type="text/javascript">
+        var OriginTitile=document.title;
```
Comment (sli1989) [inline layout/_layout.swig]: personalized too much...😅

**Comment by ehlxr** [comment]
demo page http://ehlxr.me

**Comment by wafer-li** [comment]
And I want to denote that:

image

The gitalk need **read and write** permission for **ALL** my public repos.

If you modify some gitalk script, and all my public repo will be ruin.

To get out of the posibile problem, I **personally** strongly deprecated **gitalk** and **gitment** and something like them.

------

But I DO like the other feature you provide, like the **click to show disqus**, maybe we could extract it from this PR.

**Comment by ivan-nginx** [comment]
Please, don't need to add commits with mixins features. Separate fetautes and divide it on different commits/pulls is the best choise to close all questions.

Also, pull on GiTalk aws added in #58.

**Comment by ehlxr** [comment]
ok, i will reorganize the pr

File: layout/_layout.swig (hunk 1 above)
Comment (ApolloZhu) [reply layout/_layout.swig]: @ivan-nginx Can we consider adding this feature (change to a different title when window is no longer in focus) to the list #67 ?

Comment (stevenjoezhang) [reply layout/_layout.swig]: @ApolloZhu @ivan-nginx https://github.com/theme-next/hexo-next-title

File: layout/_macro/post.swig [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {% macro render(post, is_index, post_extra_class) %}
 
-  {% set headlessPost = Array.prototype.indexOf.call(['quote', 'picture'], post.type) > -1 %}
+  {% set headlessPost = Array.prototype.indexOf.call(['quote', 'picture', 'nohead'], post.type) > -1 %}
```
Comment (stevenjoezhang) [inline layout/_macro/post.swig]: `header: false` option added in https://github.com/theme-next/hexo-theme-next/commit/92fdf867bf8d1ae146d318cfd5f134c26bae7671#diff-0e030eb6dd7b9d25bfcba06bc8ea5609R27

**Pull request closed without merging**